back-laravel/database/migrations/2024_04_15_224206_create_event_table.php

```diff
@@ -13,8 +13,7 @@ public function up(): void
     {
         Schema::create('events', function (Blueprint $table) {
             $table->id();
-            $table->unsignedBigInteger('user_id');
```

Comment:
A nível de projeto em andamento, não é bom alterar migration diretamente no arquivo.
Faz isso apenas quando não tiver subido nada pra outros desenvolvedores, pois pode causar alguns conflitos de dados no banco.
Ex: Vc trocou a coluna 'nome' pra 'name', no meu PC vai estar 'nome', pode dar conflito.
Quando for assim, cria uma migration nova. Fica poluído, mas pelo menos não fica inconsistente.